### PR TITLE
Add l10n switcher

### PR DESCRIPTION
**Pull request by michelle**
r? @jez 

To check out the result: `git checkout gh-pages; git pull` then serve the directory. You can also view it here: https://github.com/stripe/elements-examples/tree/gh-pages

screen shot 2017-11-05 at 4 49 00 pm
screen shot 2017-11-05 at 4 49 19 pm
screen shot 2017-11-05 at 4 48 49 pm

### CODE REVIEW

**Comment by michelle** [comment]
Next steps:
- [ ] Make `gh-pages` the default Github pages branch!
- [ ] Possibly add auto-detection based on `navigator.language`

**Comment by jez** [comment]
> Make gh-pages the default Github pages branch!

We're not going to have two branches, `master` and `gh-pages` going forward, right? We should just have one I think. Doesn't matter to me which one we have.

**Comment by michelle** [comment]
@jez 

OOC, why not?

**Comment by jez-stripe** [comment]
It's easy to commit to the wrong branch. Especially when there are already
people who have the repo cloned locally, chances are they'll just continue
to use `master`.

What's the current benefit to having multiple branches?

File: js/l10n.js [hunk 1]
```diff
@@ -0,0 +1,19 @@
+// Simple localization
+const isGithubPages = window.location.hostname === 'stripe.github.io';
+const localeIndex = isGithubPages ? 2 : 1;
+window.__exampleLocale = window.location.pathname.split('/')[localeIndex] || 'en';
+
+document.querySelectorAll('.optionList a').forEach(function(langNode) {
+  const langValue = langNode.getAttribute('data-lang');
+
+  const urlPrefix = isGithubPages ? '/elements-examples/' : '/';
```
Comment (jez) [inline js/l10n.js]: This could be hoisted out of the loop (near `isGithubPages`).

Comment (jez) [inline js/l10n.js]: This could be hoisted out of the loop (near `isGithubPages`).

**Pull request merged**